### PR TITLE
refactor: update copilot related templates

### PR DESCRIPTION
**Pull request by huimiu**

### CODE REVIEW

File: templates/common/copilot-gpt-basic/appPackage/declarativeCopilot.json.tpl [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
-    "$schema": "https://graphdevxdata.blob.core.windows.net/data/schemas/CopilotGPTManifestSchema-1.0.json",
-    "name": "Template declarative copilot",
-    "description": "This a template declarative copilot.",
-    "instructions": "You are a template declarative copilot. You should answer questions by reminding the user with \"I'm a template declarative copilot\"."
-}
```
Comment (sebastienlevert) [inline templates/common/copilot-gpt-basic/appPackage/declarativeCopilot.json.tpl:6]:
```json
{
    "name": "Teams Toolkit declarative copilot",
    "description": "Declarative copilot created with Teams Toolkit",
    "instructions": "You are a declarative copilot and were created with Team Toolkit. You should start every response and answer to the user with \"Thanks for using Teams Toolkit to create your declarative copilot!\\n\" and then answer the questions and help the user."
}
```

File: templates/js/copilot-gpt-from-scratch-plugin/appPackage/repairDeclarativeCopilot.json.tpl [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-            "text": "Show repair records assigned to Karin Blair"
-        }
-    ],
-    "actions": [
-        {
-            "id": "repairPlugin",
-            "file": "ai-plugin.json"
-        }
-    ]
-}
```
Comment (sebastienlevert) [inline templates/js/copilot-gpt-from-scratch-plugin/appPackage/repairDeclarativeCopilot.json.tpl:17]: `ai-plugin.json`

```json
{
  "schema_version": "v2.1",
  "namespace": "repairs",
  "name_for_human": "ttk-plugin-copilot${{APP_NAME_SUFFIX}}",
  "description_for_human": "Track your repair records",
  "description_for_model": "Plugin for searching a repair list, you can search by who's assigned to the repair.",
  "functions": [
    {
      "name": "listRepairs",
      "description": "Returns a list of repairs with their details and images",
      "parameters": {
        "type": "object",
        "properties": {
          "assignedTo": {
            "type": "string",
            "description": "The person assigned to the repair"
          }
        }
      }
    }    
  ],
  "runtimes": [
    {
      "type": "OpenApi",
      "auth": {
        "type": "None"
      },
      "spec": {
        "url": "apiSpecificationFile/repair.yml",
        "progress_style": "ShowUsageWithInputAndOutput"
      },
      "run_for_functions": ["listRepairs"]
    }
  ]
}
```

`repair.yml`

```yml
openapi: 3.0.0
info:
  title: Repair Service
  description: A simple service to manage repairs
  version: 1.0.0
servers:
  - url: ${{OPENAPI_SERVER_URL}}/api
    description: The repair api server
paths:
  /repairs:
    get:
      operationId: listRepairs
      summary: List all repairs
      description: Returns a list of repairs with their details and images
      parameters:
        - name: assignedTo
          in: query
          description: Filter repairs by who they're assigned to
          schema:
            type: string
          required: false
      responses:
        '200':
          description: A list of repairs
          content:
            application/json:
              schema:
                type: array
                items:
                  properties:
                    id:
                      type: string
                      description: The unique identifier of the repair
                    title:
                      type: string
                      description: The short summary of the repair
                    description:
                      type: string
                      description: The detailed description of the repair
                    assignedTo:
                      type: string
                      description: The user who is responsible for the repair
                    date:
                      type: string
                      format: date-time
                      description: The date and time when the repair is scheduled or completed
                    image:
                      type: string
                      format: uri
                      description: The URL of the image of the item to be repaired or the repair process
```

`repair.ts` should probably become `repairs.ts`

```ts
/* This code sample provides a starter kit to implement server side logic for your Teams App in TypeScript,
 * refer to https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference for complete Azure Functions
 * developer guide.
 */

import { app, HttpRequest, HttpResponseInit, InvocationContext } from "@azure/functions";

import repairRecords from "../repairsData.json";

/**
 * This function handles the HTTP request and returns the repair information.
 *
 * @param {HttpRequest} req - The HTTP request.
 * @param {InvocationContext} context - The Azure Functions context object.
 * @returns {Promise<Response>} - A promise that resolves with the HTTP response containing the repair information.
 */
export async function repairs(
  req: HttpRequest,
  context: InvocationContext
): Promise<HttpResponseInit> {
  context.log("HTTP trigger function processed a request.");

  // Initialize response.
  const res: HttpResponseInit = {
    status: 200,
    jsonBody: {
      results: repairRecords,
    },
  };

  // Get the assignedTo query parameter.
  const assignedTo = req.query.get("assignedTo");

  // If the assignedTo query parameter is not provided, return the response.
  if (!assignedTo) {
    return res;
  }

  // Filter the repair information by the assignedTo query parameter.
  const repairs = repairRecords.filter((item) => {
    const fullName = item.assignedTo.toLowerCase();
    const query = assignedTo.trim().toLowerCase();
    const [firstName, lastName] = fullName.split(" ");
    return fullName === query || firstName === query || lastName === query;
  });

  // Return filtered repair records, or an empty array if no records were found.
  res.jsonBody.results = repairs ?? [];
  return res;
}

app.http("repairs", {
  methods: ["GET"],
  authLevel: "anonymous",
  handler: repairs,
});
```

File: templates/common/copilot-gpt-basic/README.md [hunk 3]
```diff
@@ -1,48 +0,0 @@
-# Overview of the Basic Declarative Copilot template
```
Comment (sebastienlevert) [inline templates/common/copilot-gpt-basic/README.md:1]:
```md
# Overview of the basic declarative copilot template
```

File: templates/common/copilot-gpt-basic/README.md [hunk 4]
```diff
@@ -1,48 +0,0 @@
...
-
-## Build a basic declarative copilot
-
-With the declative copilot, you can build a custom version of Copilot that can be used for specific scenarios, such as for specialized knowledge, implementing specific processes, or simply to save time by reusing a set of AI prompts. For example, a grocery shopping Copilot GPT can be used to create a grocery list based on a meal plan that you send to the Copilot GPT.
```
Comment (sebastienlevert) [inline templates/common/copilot-gpt-basic/README.md:5]:
```md
With the declarative copilot, you can build a custom version of Copilot that can be used for specific scenarios, such as for specialized knowledge, implementing specific processes, or simply to save time by reusing a set of AI prompts. For example, a grocery shopping Copilot declarative copilot can be used to create a grocery list based on a meal plan that you send to Copilot.
```

File: templates/js/copilot-gpt-from-scratch-plugin/README.md [hunk 5]
```diff
@@ -1,54 +0,0 @@
-# Overview of the Declarative Copilot template
```
Comment (sebastienlevert) [inline templates/js/copilot-gpt-from-scratch-plugin/README.md:1]:
```md
# Overview of the declarative copilot template
```

File: templates/js/copilot-gpt-from-scratch-plugin/README.md [hunk 6]
```diff
@@ -1,54 +0,0 @@
...
-
-## Build a declarative copilot from a new API with Azure Functions
-
-With the declarative copilot, you can build a custom version of copilot that can be used for specific scenarios, such as for specialized knowledge, implementing specific processes, or simply to save time by reusing a set of AI prompts. For example, a grocery shopping declarative copilot can be used to create a grocery list based on a meal plan that you send to the declarative copilot.
```
Comment (sebastienlevert) [inline templates/js/copilot-gpt-from-scratch-plugin/README.md:5]:
```md
With the declarative copilot, you can build a custom version of copilot that can be used for specific scenarios, such as for specialized knowledge, implementing specific processes, or simply to save time by reusing a set of AI prompts. For example, a grocery shopping declarative copilot can be used to create a grocery list based on a meal API that you integrate with your declarative copilot.
```

**Review by hund030 — APPROVED** [review]

**Pull request merged**